Lib/ctypes/__init__.py

```diff
@@ -263,6 +263,8 @@ class c_void_p(_SimpleCData):
 
 class c_bool(_SimpleCData):
     _type_ = "?"
+c_bool.__ctype_le__ = c_bool.__ctype_be__ = c_bool
```

Comment:
Could you add a comment above this like `GH-127280: brief description of the issue` so people in the future know why it's there?

Reply:
I'm not sure I understand the need for the comment here. None of the other primitive types with this same pattern have such a comment.

Reply:
Ah yeah, I wasn't looking above. I think it's fine to leave it then.